Cargo.toml

```diff
@@ -36,7 +36,7 @@ product-name = "Space Acres"
 
 [dependencies]
 anyhow = "1.0.80"
-betrayer = {version = "0.1.0"}
+betrayer = { git = "https://github.com/hunjixin/betrayer.git", rev="d355b732e4e3a13a08148fc8ff3f9d247b4001fc" }                             
```

Comment:
Please don't add git revisions from your forks, it will not be merged. Only upstream git revisions are accepted and even then they require comments explaining why it is necessary and when can be switched to upstream release.

Also you have a ton of whitespaces at the end of the line here.

Reply:
kde and ubuntu button style not the same,  and wait for betrayer next release